src/main/java/com/team01/carsharingapp/telegramapi/Bot.java

```diff
@@ -0,0 +1,46 @@
+package com.team01.carsharingapp.telegramapi;
+
+import com.team01.carsharingapp.exception.TelegramException;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+public class Bot extends TelegramLongPollingBot {
+    private final String botUsername;
+    private final Long chatId;
+
+    public Bot(String botUsername, String botToken, Long chatId) {
+        super(botToken);
+        this.botUsername = botUsername;
+        this.chatId = chatId;
+    }
+
+    @Override
+    public void onUpdateReceived(Update update) {
+        if (update.hasMessage() && update.getMessage().getText().equals("/start")) {
+            sendMessage("Hello mate");
+        } else {
+            sendMessage(
+                    "I'm here not to speak with u, i just send u notification when they arrived"
```

Comment:
Maybe change it to something like “Unknown command” for now. Or disable reading messages at all.